README.md

```diff
@@ -99,6 +99,123 @@ Run the following command:
 docker-compose up -d
 ```
 
+#### Docker Swarm w/ Docker Secrets:
+
+This section discusses running an instance of ibeam inside a locked docker swarm, and using the docker swarm facilities for managing secrets.
+Please refer to the following articles for in-depth details on docker swarm, locking the swarm, and using docker secrets.
+
+- [Swarm mode overview](https://docs.docker.com/engine/swarm/)
+- [Lock your swarm to protect its encryption key](https://docs.docker.com/engine/swarm/swarm_manager_locking/)
+- [Manage sensitive data with Docker secrets](https://docs.docker.com/engine/swarm/secrets/)
+
+Once you have a locked docker swarm instance initialized, you can create the secrets.
+On your host system create two secure (meaning not world-readable) files containing your Interactive Brokers account name and password:
+
+1. ib.account.txt
+2. ib.password.txt
+
+Next, inject these secrets into the docker swarm by using `docker secret create`:
+
+```posh
+docker secret create IBEAM_ACCOUNT_v1 ib.account.txt
+docker secret create IBEAM_PASSWORD_v1 ib.password.txt
+```
+
+Once you've initialized the secrets you should remove the original files from your host system.
```

Comment:
How crucial is this? Is not doing so a big no-no? If so, should we highlight it more and change `should` to a more actionable phrase like `Once you've initialized the secrets remove the original...`

Reply:
Failing to remove those files leaves unencrypted secrets on the disk, so I'll change it to an instruction:

```
Once you've initialized the secrets delete the original files from your host system.
```